R/args.R

```diff
@@ -89,9 +91,9 @@ CmdStanArgs <- R6::R6Class(
         random = TRUE
       )
     },
-    new_files = function(type = c("output", "diagnostic")) {
+    new_files = function(type = c("output", "diagnostic", "profile")) {
       files <- file.path(self$output_dir, self$new_file_names(type))
-      invisible(file.create(files))
+      if (type != "profile") invisible(file.create(files))
```

Comment:
Just curious why the difference here for the profile files

Reply:
Because this makes the check whether there was profiling CSV data generated simpler. We just check whether files exist, not if they are empty.

I honestly think we dont need to create any of the files as cmdstan will create them if they dont exits.

Reply:
Ok, it's fine by me to not create any of the files if cmdstan will do it